lld/COFF/Driver.cpp

```diff
@@ -2039,6 +2039,7 @@ void LinkerDriver::linkerMain(ArrayRef<const char *> argsArr) {
     parseMerge(".ctors=.rdata");
     parseMerge(".dtors=.rdata");
     parseMerge(".CRT=.rdata");
+    parseMerge(".data_cygwin_nocopy=.data");
```

Comment:
This bit looks to be somewhat disjoint from the rest of the change here (also for what's outlined in the commit message).

Reply:
Would I need a separate PR or could I just describe it in the commit message?

Reply:
no, it makes sense to test `.data_cygwin_nocopy` along with `__data_start__` and `__data_end__`, because the whole point of it is that it is outside (after) those symbols.

Reply:
Oh, I see. Then in that case it totally makes sense to include it in this patch and test both at the same time!